pattern-1/dockerfiles/mysql/build.sh

```diff
@@ -18,19 +18,21 @@
 #
 # ------------------------------------------------------------------------
 
+set -e
+
 # Product Context
 product=wso2ei
 productVersion=6.1.1
 patternNumber=1
 # MySQL parameters
-rdbms=mysql
+database=mysql
 sqlVersion=5.5
 # Container Cluster Manager Info
 platform=kubernetes
 # Image Info
-repository=${product}-${platform}-pattern${patternNumber}-${rdbms}
+repository=${product}-${platform}-pattern${patternNumber}-${database}
 tag=${sqlVersion}
 
 # Source
-echo "Creating ${rdbms} database for pattern ${patternNumber} ..."
+echo "Creating ${database} database for pattern ${patternNumber} ..."
```

Comment:
Shall we use the same, echo "Creating ${repository}:${tag}..." here as well ?

Reply:
+1 Thanks for pointing! Will update!